bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/OnlineOfflineType.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.library.types;
+
+import org.eclipse.smarthome.core.library.internal.StateConverterUtil;
+import org.eclipse.smarthome.core.types.Convertible;
+import org.eclipse.smarthome.core.types.PrimitiveType;
+import org.eclipse.smarthome.core.types.State;
+
+public enum OnlineOfflineType implements PrimitiveType, State, Convertible {
```

Comment:
ONLINE/OFFLINE must not be introduced as a new state - it is a "status" which only applies for Things, not for Items.